CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/font-preferences.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Font Preferences</title>
+        <link rel="stylesheet" href="nbdocs://au.gov.asd.tac.constellation.preferences/au/gov/asd/tac/constellation/preferences/constellation.css" type="text/css">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    </head>
+    <body>
+        <h1>Font Preferences</h1>       
+        <p>
+            Font preferences can be accessed via Setup -&gt; Options -&gt; CONSTELLATION -&gt; Font.
```

Comment:
The name of the tab from what I could see was "Application Font" rather than "Font" so this should probably be updated here and in other places in the help (such as table of contents, title of page, etc.) just to avoid confusion.